src/ai/backend/common/events/hub/propagators/bypass.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
+import logging
+import uuid
+from typing import AsyncIterator, Optional
+
+from ai.backend.common.events.dispatcher import AbstractEvent
+from ai.backend.logging.utils import BraceStyleAdapter
+
+from ..hub import EventPropagator
+
+log = BraceStyleAdapter(logging.getLogger(__spec__.name))
+
+
+class AsyncBypassPropagator(EventPropagator):
```

Comment:
I don't understand what it is bypassing. 
It seems like it might be bypassing the last finished event fetcher, but it's too hard to infer just from the name.